### ISSUE TITLE
Release on ROS Lunar

### ISSUE THREAD

**Issue by mikaelarguedas** [OctoMap/octomap#165]
Hi @ahornung and Octomap maintainers!

As you may know the next ROS release Lunar Loggerhead is around the corner :tada: 
Is it possible to roll out the current release of the Octomap stack on ROS Lunar? Being a low level set of packages it will [allow many maintainers](http://repositories.ros.org/status_page/blocked_releases_lunar.html?q=id%3D%22octomap%22&s=6&r=1) to release their own packaages the depend directly or indirectly on octomap.

Thanks!

**Comment by ahornung** [comment]
Thanks for the heads up! I think the best time to introduce a new version would be now. I am planning a new release 1.9 based on the `devel` branch, this new version would be ideal to release into Lunar now.

However, I am a bit too busy right now for intensive maintenance. Would it be possible to support, e.g. by creating the devel jobs as a first step for Lunar?

**Comment by mikaelarguedas** [comment]
Sure, new distro is the best time to try out some changes!
We do recommand to release the current version with a disclaimer that a new released is coming and ABI may break. This way people downstream packages [are not blocked](http://repositories.ros.org/status_page/blocked_releases_lunar.html?q=label%3D%22UNBLOCKED_BLOCKING%22&s=6&r=1), helping us to move the release of `desktop_full` forward; with the knowledge that this is still testing phase and future work may be required before the official release. (e.g. that is what have been done for orocos https://github.com/ros/rosdistro/pull/14365)

**Comment by ahornung** [comment]
In that case, sure, I can go ahead and release the current version first.

**Comment by mikaelarguedas** [comment]
Awesome, thanks!

**Comment by mikaelarguedas** [comment]
Hi @ahornung!
We will enter the beta testing phase of ROS Lunar at the end of the week. We try not to have any API breakage after that point. Do you have any ETA for testing and landing octomap 1.9 in Lunar ?

BTW: perception_pcl should get in tomorrow, so every octomap package should have their dependencies satisfied by then.

Cheers

**Comment by ahornung** [comment]
Thanks for the heads up! I'll try to have 1.9 ready by the weekend.

**Comment by mikaelarguedas** [comment]
Thanks @ahornung  for releasing 1.9, looks like everything built fine :tada: 
I'm keeping this open because we're still waiting on `octomap_ros` and `octomap_mappig` to be released. Feel free to close this and open a Lunar release issue on the corresponding repos if you think it's better

**Comment by ahornung** [comment]
I'll close this issue as it's resolved, and release octomap_ros asap.

However, I'm no longer actively working on octomap_mapping and also don't have time to thoroughly test it. So if someone needs it released then I need support / testing feedback.

**Comment by mikaelarguedas** [comment]
> I'll close this issue as it's resolved, and release octomap_ros asap.

:+1: 

> However, I'm no longer actively working on octomap_mapping and also don't have time to thoroughly test it. So if someone needs it released then I need support / testing feedback.

I don't know if anybody needs it right now and expect people to open an issue if they need it, so no rush for testing/releasing. Thanks again for the iterations on releasing octomap!
